readme.md

```diff
@@ -2,12 +2,10 @@
 
 <div align="center">
 
-  <h3><b>Bookstore React App</b></h3>
+  <h3><b>Bookstore React App V2</b></h3>
```

Comment:
- [ ] Great job on your mileston ,i would suggest you to make sure your  final design should match Zeplin's design (this includes all elements + the same styling) ,kindly make your design align on the same line kust like the zeplin given design as this willl make sure project look professional and user friendly ,see screenshot below to compare 
- [ ] zeplin design
![Screenshot 2023-09-28 072316](https://github.com/Rifat8080/Bookstore-React-App/assets/92458236/4ab50584-5ed3-43a2-bec6-98ca201ef959)


- [ ] Your design
![Screenshot 2023-09-28 073451](https://github.com/Rifat8080/Bookstore-React-App/assets/92458236/d92207c6-67c6-41e8-9708-ba9c477b7b29)


- [ ] make sure your designstyling look exactly like this original zeplin design
![bookstore](https://github.com/Rifat8080/Bookstore-React-App/assets/92458236/6cccdaff-de7b-4a72-b35b-17bdc77e1f30)

Reply:
- @Rifat8080 as requested by the previous reviewer in this [review link](https://github.com/Rifat8080/Bookstore-React-App/pull/10#discussion_r1339606408), you are yet to match the original design provided by zeplin and i will kindly suggest that you make the appropriate design corrections so that your project can be approved